### DISCUSSION TITLE
Criar Rotas usando Laravel

### DISCUSSION THREAD

**Discussion by Fabrowser** [frontendbr/forum#2218] [Perguntas e Respostas]
Boa noite pessoal tudo bem?

Nao estou conseguindo criar um controlador de CRUD no laravel 8 (A versão do laravel do tutorial que estou seguindo é a 5.5 , ou seja diferente da minha,, talvez seja isso o problema , mas nao achei nada na internet sobre isso. por isso estou pedindo a ajuda de vocês)

O que eu fiz:

1 - ja criei o Controle através do shell do windows:

php artisan make:controller Admin/ArtigosController --resource

2 - Em seguida Vou na pasta de routes/web.php e crio as Rotas:

Route::get('/home', [App\Http\Controllers\HomeController::class, 'index'])->name('home'); (esta linha ja veio no arquivo do Laravel. Coloquei apenas as duas linhas abaixo no arquivo)

Route::middleware(['auth']) -> prefix('admin') -> namespace('Admin') -> group(function(){

Route::resource ('artigos','ArtigosController');

});

3 - Em seguida eu deveria digitar o seguinte endereço no navegador: http://localhost:8000/admin/artigos para exibir a mensagem de "ok" (método no index do Controller que eu criei **return "ok" **) , mas em vez disso, aparece a mensagem abaixo, conforme podem ver:Boa noite pessoal tudo bem?

Nao estou conseguindo criar um controlador de CRUD no laravel 8 (A versão do laravel do tutorial que estou seguindo é a 5.5 , ou seja diferente da minha,, talvez seja isso o problema , mas nao achei nada na internet sobre isso. por isso estou pedindo a ajuda de vocês)

O que eu fiz:

1 - ja criei o CRUD atraves do shell do windows:

php artisan make:controller Admin/ArtigosController --resource

2 - Em seguida Vou na pasta de routes/web.php e crio as Rotas:

Route::get('/home', [App\Http\Controllers\HomeController::class, 'index'])->name('home'); (esta linha ja veio no arquivo do Laravel. Coloquei apenas as duas linhas abaixo no arquivo)

Route::middleware(['auth']) -> prefix('admin') -> namespace('Admin') -> group(function(){

Route::resource ('artigos','ArtigosController');

});

3 - Em seguida eu deveria digitar o seguinte endereço no navegador: http://localhost:8000/admin/artigos para exibir a mensagem de "ok" (método no index do Controller que eu criei **return "ok" **) , mas em vez disso, aparece a mensagem abaixo, conforme podem ver:

![image](https://user-images.githubusercontent.com/79614829/146693198-41a67e84-a412-4297-bc29-00a2ed40f8db.png)

**Será que é a versão do Laravel? Será que as rotas devem ser definidas de outra forma com a versão 8? Se puderem me ajudar agradeço muito.**

**Comment by henriquedev00** [comment]
Tente inserir o comando "use App\Http\Controllers\Admin\ArtigosController" no topo do seu arquivo web na pasta routes. (insira o comando sem as aspas e após a abertura do codigo php <?php)
